src/api/IBMiContent.ts

```diff
@@ -649,6 +651,32 @@ export default class IBMiContent {
     return items;
   }
 
+  async memberResolveLibraryList(member: string, file: string, libraryList?: ILELibrarySettings): Promise<IBMiMember | undefined> {
+    if (this.ibmi.remoteFeatures[`GETMBR.PGM`]) {
+      let env: {[id: string]: string} = {};
+
+      if (libraryList) {
+        if (libraryList.currentLibrary) env[`&CURLIB`] = libraryList.currentLibrary;
+        if (libraryList.currentLibrary) env[`&LIBL`] = libraryList.libraryList.join(` `);
```

Comment:
Shouldn't this be `libraryList.libraryList`?

```suggestion
        if (libraryList.libraryList) env[`&LIBL`] = libraryList.libraryList.join(` `);
```